README.md

```diff
@@ -139,21 +139,36 @@ chmod 400 /path/to/phone_buddy_key_pair.pem
 
 As of January 2023, we're having weekly meetings to check in on development and move this project along. If you're interested in helping out, send Teo a WhatsApp at +33 6 17 50 71 28. 
 
+
+## Team Meeting Time
+
+All team members are expected to attend.
+
+**Meeting Time**: Fridays, 9:30 - 10:30 PM Afghanistan Time (UTC+4:30)
```

Comment:
Could you list the meeting time in Pacific Time please? Officially I'm the creator of the meeting, so if daylight savings changes things in Kabul it might actually be at a different time. I'll try to remember to tell you if things change there, but I have a feeling that might pose a problem 😅. 

We could link to this, which I think could be helpful: https://timezonewizard.com/gv-jsd

Reply:
I'm aware that you organize the meeting, and for that reason, I've reached out for your feedback and any suggestions you might have. Your insights are particularly valuable given your role in setting up this meeting. I appreciate your consideration and look forward to any recommendations :)